javascript/javascript3/week2/promise-visual homework folder/main.js

```diff
@@ -0,0 +1,57 @@
+const redCircle = document.querySelector('ul.marks li:nth-child(1)');
+const blueCircle = document.querySelector('ul.marks li:nth-child(2)');
+const greenCircle = document.querySelector('ul.marks li:nth-child(3)');
+
+// translateOneByOne function 
+
+function translateOneByOne() {
```

Comment:
I am not sure but i think it will not display once at the time because the promises will be resolved in mile second
so you need to use setTimeOut()